lib/tools/mp4-inspector.js

```diff
@@ -425,6 +425,24 @@ var
         boxes: inspectMp4(data)
       };
     },
+    ctts: function(data) {
+      var
+        view = new DataView(data.buffer, data.byteOffset, data.byteLength),
+        result = {
+          version: view.getUint8(0),
+          flags: new Uint8Array(data.subarray(1, 4)),
+          compositionOffsets: []
+        },
+        entryCount = view.getUint32(4),
+        i;
+      for (i = 8; entryCount; i += 8, entryCount--) {
+        result.compositionOffsets.push({
+          sampleCount: view.getUint32(i),
+          sampleOffset: view.getInt32(i + 4)
```

Comment:
in the spec for mp4 `version 0` of this box uses an `uint` while `version 1` uses an int. Would you mind making the change for this case?

Reply:
can you please give me a reference for ctts spec?
[bento4](https://github.com/axiomatic-systems/Bento4/blob/be15647cc9750489cb2972f630d099b6426f03a7/Source/C%2B%2B/Core/Ap4CttsAtom.cpp#L102) actually uses only uint and used to have checks for quick time support
as far as i understand negative offsets ware used only in quicktime and they ware never part of the spec